docs/install.md

```diff
@@ -5,38 +5,28 @@ post installation to ensure the operator can function correctly.
 
 ## STS Clusters
 
-### Post operator installation
+### Pre operator installation
```

Comment:
Those are the instructions post the installation. The pre installation instructions are in the [prerequisites.md](https://github.com/openshift/aws-load-balancer-operator/blob/main/docs/prerequisites.md).